packages/web/src/components/Icon/Icon.module.css

```diff
@@ -27,3 +27,6 @@
 .xxLarge {
   --icon-size: var(--unit-8);
 }
+.xxxLarge {
```

Comment:
I believe this exists in main now. I did the same thing in my last PR

Reply:
hm just merged main and didn't see it